### ISSUE TITLE
SplitDateTimeField doesn't show errors

### ISSUE THREAD

**Issue by simplylizz** [django-crispy-forms/django-crispy-forms#1065]
* Package version: 1.9.2
* Django version: 3.1.1
* Python version: 3
* Template pack: bootstrap4

### Description:

`SplitDateTimeField` doesn't show errors. E.g. when only date is filled and time input is empty.

**Comment by smithdc1** [comment]
Yes, this is related to existing issue with multifields with this template pack.

There is a tentative patch for this but I'm fairly sure that is the wrong way to fix it. #976 

Any other ideas welcomed.

**Comment by simplylizz** [comment]
https://github.com/simplylizz/django-crispy-forms/commit/3c8403541cb9cbbb2568a03ef9a4929c0437d8e2 - I'm going to stick with this workaround, for my case (building a prototype) it's enough. Yes, it will highlight all `MultiWidget` fields in case of error, though I think it's much better than a completely hidden error.

Also don't know if it will break any other use cases, I'm not a frontend-guy.

**Comment by edu2004eu** [comment]
For anyone looking for a quick, unintrusive workaround, you can simply add `display: block` to the error container.

    #error_0_id_<field_name>, #error_1_id_<field_name> {
        display: block;
    }

The container is only rendered if there's an error, so it wouldn't override anything unexpected.
